astropy/io/fits/tests/test_diff.py

```diff
@@ -893,3 +893,34 @@ def test_fitsdiff_with_names(tmpdir):
 
     diff = FITSDiff(path1, path2)
     assert "Extension HDU 1:" in diff.report()
+
+
+def test_rawdatadiff_diff_with_rtol(tmpdir):
+    """Regression test for https://github.com/astropy/astropy/issues/13330"""
+    path1 = str(tmpdir.join("test1.fits"))
+    path2 = str(tmpdir.join("test2.fits"))
+    a = np.zeros((10, 2), dtype='float32')
+    a[:, 0] = np.arange(10, dtype='float32') + 10
+    a[:, 1] = np.arange(10, dtype='float32') + 20
+    b = a.copy()
+    changes = [(3, 13.1, 23.1), (8, 20.5, 30.5)]
+    for i, v, w in changes:
+        b[i, 0] = v
+        b[i, 1] = w
+
+    ca = Column('A', format='20E', array=[a])
+    cb = Column('A', format='20E', array=[b])
+    hdu_a = BinTableHDU.from_columns([ca])
+    hdu_a.writeto(path1, overwrite=True)
+    hdu_b = BinTableHDU.from_columns([cb])
+    hdu_b.writeto(path2, overwrite=True)
+    with fits.open(path1) as fits1:
+        with fits.open(path2) as fits2:
+
+            diff = FITSDiff(fits1, fits2, atol=0, rtol=0.001)
+            str1 = diff.report(fileobj=None, indent=0)
+
+            diff = FITSDiff(fits1, fits2, atol=0, rtol=0.01)
+            str2 = diff.report(fileobj=None, indent=0)
+
+    assert str1.split("Extension HDU 1:")[1] != str2.split("Extension HDU 1:")[1]
```

Comment:
Could you test explicitly the output ? Here it's not clear what to expect, I guess str2 should report differences and not str1 ?